Microsoft.Toolkit.Uwp.UI.Controls/RangeSelector/RangeSelector.cs

```diff
@@ -195,10 +195,12 @@ private void MinThumb_KeyDown(object sender, KeyRoutedEventArgs e)
             {
                 case VirtualKey.Left:
                     RangeMin -= 1;
+                    SyncThumbs();
```

Comment:
How does this work with StepFrequency set to 10?
Should we adjust the thumbs when the RangeMin and RangeMax change?

Reply:
I tried it, but if you adjust thumbs on range min/max value change you lose the "smooth dragging".